setup.py

```diff
@@ -146,7 +146,7 @@
     "networkx.algorithms.tree": ["tests/*.py"],
     "networkx.classes": ["tests/*.py"],
     "networkx.generators": ["tests/*.py", "atlas.dat.gz"],
-    "networkx.drawing": ["tests/*.py"],
+    "networkx.drawing": ["tests/*.py", "tests/baseline/*png"],
```

Comment:
Yes, I added it to the `MANIFEST`.  At this point, it is only going to add a few KB.  I am also concerned about the repo size, so we will need to be very careful what we add.  I will add a note to the `CONTRIBUTOR` guide.  They provide a hash option, I believe, to avoid having to add the images to the repo.  We should investigate that after the 2.7 release (and before the 3.0 release).

Reply:
I've added text to the CONTRIBUTING guide.  Once 2.7 is released, I will investigate using the hash library and maybe storing the images in an external repository.